README.md

```diff
@@ -102,7 +102,7 @@ and others would not be possible without your help.
 ## Online Documentation
 
 Since Kyuubi 1.3.0-incubating, the Kyuubi online documentation is hosted by [https://kyuubi.apache.org/](https://kyuubi.apache.org/).
-You can find the latest Kyuubi documentation on [this web page](https://kyuubi.apache.org/docs/latest/).
+You can find the latest Kyuubi documentation on [this web page](https://kyuubi.readthedocs.io/en/master/).
 For 1.2 and earlier versions, please check the [Readthedocs](https://kyuubi.readthedocs.io/en/v1.2.0/) directly.
 
 ## Quick Start
```

Comment:
the link of `Getting Started` is unavailable, could you update it as well?

Reply:
done. @cfmcgrady